### PR TITLE
Keep one admin on orgs

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/orgs/views.py [hunk 1]
```diff
@@ -1610,13 +1611,20 @@ def add_per_invite_fields(self, org: Org):
 
             def clean_invite_emails(self):
                 emails = self.cleaned_data["invite_emails"].lower().strip()
+                existing_users_emails = list(self.org.get_users().values_list("username", flat=True))
```
Comment (rowanseymour) [inline temba/orgs/views.py:1614]: might as well be `set(...)`

Comment (rowanseymour) [reply temba/orgs/views.py:1614]: should also check there aren't existing invitations for these emails

File: temba/orgs/views.py [hunk 2]
```diff
@@ -2115,6 +2123,15 @@ def pre_process(self, request, *args, **kwargs):  # pragma: needs cover
                 logout(request)
                 return HttpResponseRedirect(reverse("orgs.org_join", args=[secret]))
 
+            if not org.get_admins().exclude(id=request.user.id).exists():
```
Comment (rowanseymour) [inline temba/orgs/views.py:2126]: is this even necessary with the above checks ? Or shouldn't it more general - you can't join a workspace you already belong to you.

**Review by rowanseymour — CHANGES_REQUESTED** [review]

File: temba/orgs/views.py (hunk 1 above)
Comment (norkans7) [reply temba/orgs/views.py:1614]: we currently allow admin to kind of modify the role on the invite sent already so the invited user get multiple emails but with a consistent link so which ever the click they join with the latest role set.

I think by forcing admin to remove the invite and add a new one that causes more confusion.
We can keep current approach

Comment (rowanseymour) [reply temba/orgs/views.py:1614]: I dunno.. what you just described sounds sounds a lot more confusing to me than just saying you can't invite the same email more than once. Can't think of a good reason why we need to support that.

**Pull request merged**